fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -133,25 +134,25 @@ private fun privateModeAdapterItems() = listOf(AdapterItem.PrivateBrowsingDescri
 private fun onboardingAdapterItems(
     onboardingState: OnboardingState,
     onboardingConfig: OnboardingConfig,
-): List<AdapterItem> {
-    val items: MutableList<AdapterItem> = mutableListOf(AdapterItem.OnboardingHeader)
+): List<OnboardingAdapterItem> {
+    val items: MutableList<OnboardingAdapterItem> = mutableListOf(OnboardingAdapterItem.OnboardingHeader)
```

Comment:
this is unused? where is this going to be used? (writing this having not read parts 3-6 yet)

Reply:
Will be used in later parts.